### PR TITLE
Add connectrpc/go v1.11.0 plugin

### PR DESCRIPTION
**Pull request by pkwarren**

### CODE REVIEW

File: plugins/connectrpc/go/v1.11.0/buf.plugin.yaml [hunk 1]
```diff
@@ -0,0 +1,19 @@
+version: v1
+name: buf.build/connectrpc/go
```
Comment (pkwarren) [inline plugins/connectrpc/go/v1.11.0/buf.plugin.yaml:2]: Chose to call the plugin `connectrpc/go` for less stuttering and symmetry with `protocolbuffers/go` and `grpc/go`.

**Review by mfridman — APPROVED** [review]

File: plugins/connectrpc/go/v1.11.0/buf.plugin.yaml (hunk 1 above)
Comment (mfridman) [reply plugins/connectrpc/go/v1.11.0/buf.plugin.yaml:2]: Also parity with https://buf.build/grpc/go, so 👍

File: plugins/connectrpc/go/v1.11.0/buf.plugin.yaml [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+plugin_version: v1.11.0
+source_url: https://github.com/connectrpc/connect-go
+description: Generates client and server stubs for connect-go. Compatible with the gRPC, gRPC-Web, and Connect RPC protocols.
```
Comment (akshayjshah) [inline plugins/connectrpc/go/v1.11.0/buf.plugin.yaml:5]: Can we s/connect-go/Connect? Or go straight to `connectrpc.com/connect`?

**Review by akshayjshah — APPROVED** [review]

File: plugins/connectrpc/go/v1.11.0/buf.plugin.yaml (hunk 2 above)
Comment (pkwarren) [reply plugins/connectrpc/go/v1.11.0/buf.plugin.yaml]: Updated to `connectrpc.com/connect`.

**Pull request merged**